src/main/java/ShuntingYardCalculator/Constants.java

```diff
@@ -0,0 +1,9 @@
+package ShuntingYardCalculator;
```

Comment:
Not good for config class.
You can use properties file or if you use it one location and related only to the gui input, just type it in the start of the classs